Student_BCOs/XiyingDing_BCO.json

```diff
@@ -0,0 +1,254 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000524",
+    "checksum": "66038b32a3553b15a5c37a0217368a6fe259aed9ad6c58ae81a2134732289f10",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Detection of transcript variation by analysis of RNA-seq data using targeted clinical genomics pipeline TAP",
+        "version": "0.4.2",
+        "license": "http://creativecommons.org/licenses/by/4.0/",
+        "created": "2019-10-07T22:19:18.384004",
+        "modified": "2019-10-17T09:54:05.613143",
+        "contributors": [
+            {
+                "name": "Xiying Ding",
+                "orcid": "",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "xzd@gwu.edu"
+            },
+            {
+                "name": "Readman Chiu",
+                "orcid": "https://orcid.org/0000-0002-2215-5535",
+                "affiliation": "Canada’s Michael Smith Genome Sciences Centre, BC Cancer Agency",
+                "contribution": [
+                    "authoredBy", "derivedFrom"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Ka Ming Nip",
+                "orcid": "",
+                "affiliation": "Canada’s Michael Smith Genome Sciences Centre, BC Cancer Agency",
+                "contribution": [
+                    "authoredBy", "derivedFrom"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Justin Chu",
+                "orcid": "",
+                "affiliation": "Canada’s Michael Smith Genome Sciences Centre, BC Cancer Agency",
+                "contribution": [
+                    "authoredBy", "derivedFrom"],
+                "email": ""
+            },
+            {
+                "name": "Inanc Birol",
+                "orcid": "",
+                "affiliation": "Canada’s Michael Smith Genome Sciences Centre, BC Cancer Agency and Department of Medical Genetics, The University of British Columbia",
+                "contribution": [
+                    "authoredBy", "derivedFrom"
+                ],
+                "email": "ibirol@bcgsc.ca"
+            }
+        ]
+    },
+    "usability_domain": [
+        "TAP is an RNAseq data analysis pipeline that is superior in speed and accuracy. It uses four steps to classify and reorganize reads from data and detect the structural and splicing variants. By analyzing open data from the published researches, TAP's result is not only consistent with researches, but also has new discoveries, proving that it is a powerful pipeline."
```

Comment:
Please provide  "This BCO was adapted by [paper name] by [author list] [doi/PMID]. See #101